torch/distributions/kl.py

```diff
@@ -2,14 +2,27 @@
 from functools import total_ordering
 
 import torch
+import math
 
 from .distribution import Distribution
+from .bernoulli import Bernoulli
+from .beta import Beta
+from .dirichlet import Dirichlet
+from .exponential import Exponential
 from .gamma import Gamma
+from .gumbel import Gumbel
+from .laplace import Laplace
+from .normal import Normal
+from .pareto import Pareto
+from .uniform import Uniform
 
 _KL_REGISTRY = {}  # Source of truth mapping a few general (type, type) pairs to functions.
 _KL_MEMOIZE = {}  # Memoized version mapping many specific (type, type) pairs to functions.
 
 
+_euler_gamma = 0.57721566490153286060
```

Comment:
nit: would you mind moving all the math helpers down into the "Implementations" section? I feel like the registration logic is a bit complex, and it would be easier to understand if it is clearly separated from the math.